CMakeLists.txt

```diff
@@ -225,12 +224,12 @@ if(LOVR_ENABLE_HEADSET AND LOVR_USE_OCULUS)
   endif()
   include_directories("${LOVR_OCULUS_PATH}/LibOVR/Include")
   link_directories("${LOVR_OCULUS_PATH}/LibOVR/Lib/Windows/${OCULUS_ARCH}/${OCULUS_BUILD_TYPE}/VS2017")
-  set(LOVR_OCULUS LibOVR)
+  set(LOVR_OCULUS "${LOVR_OCULUS_PATH}/LibOVR/Lib/Windows/${OCULUS_ARCH}/${OCULUS_BUILD_TYPE}/VS2017/LibOVR.lib")
```

Comment:
Just curious about this change: It seems like either the `link_directories` or the absolute path are necessary to get this to work, but not both.  I do like the absolute path better, maybe link_directories could get removed?  Or is it needed for the dll?  Long term this should really be an imported target like the other 3rd party libs.

Reply:
I... honestly have no idea. lemme boot into windows and see if I can just remove the change.

Reply:
Removing the absolute path in set doesn't work:

```  
LINK Pass 1: command "C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1426~1.288\bin\Hostx64\x64\link.exe /nologo CMakeFiles\Alloverse.dir\src\allonet_loader.c.obj /out:Alloverse.exe /implib:Alloverse.lib /pdb:Alloverse.pdb /version:0.0 /machine:x64 /debug /INCREMENTAL /subsystem:console deps\lovr\lovr.lib deps\allonet\allonet.lib deps\lovr\luajit\src\luajit.lib deps\lovr\glfw\src\glfw3dll.lib deps\lovr\msdfgen\msdfgen.lib deps\lovr\ode\ode.lib ..\..\..\deps\lovr\deps\openvr\bin\win64\openvr_api64.lib LibOVR.lib deps\allonet\lib\enet\enet.lib deps\allonet\lib\cJSON\libcjson_utils.lib deps\allonet\lib\cJSON\libcjson.lib deps\allonet\lib\opus\opus.lib deps\allonet\lib\tinycthread\tinycthread.lib ws2_32.lib winmm.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\Alloverse.dir/intermediate.manifest CMakeFiles\Alloverse.dir/manifest.res" failed (exit code 1104) with the following output:
C:\Users\nevyn\Dev\allovisor-lovr\out\build\x64-Debug\LINK : fatal error LNK1104: cannot open file 'LibOVR.lib'```

 .. looks like most the stuff is using absolute addressing anyway, so might as well do that here too. Removing the link_directories works fine. Doing that and submitting patch.